test/index-static-singleton.js

```diff
@@ -0,0 +1,20 @@
+var indexStaticWatcher = require('../lib/index-static-watcher').create();
+
+var indexStatic;
+
+exports.get = function() {
+  if (!indexStatic) {
+    throw new Error("indexStatic has not yet been built.");
+  }
+  return indexStatic;
+};
+
+exports.build = function(done) {
+  this.timeout(10000);
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) return done(err);
+
+    indexStatic = newIndexStatic;
+    done();
+  });
+};
```

Comment:
oh yea you are right!  just did a search again and found TWOOOOO occurrence of `require('./index-static-singleton')`.  Sorry for the confusion :stuck_out_tongue_closed_eyes: